modules/javafx.graphics/src/main/java/com/sun/prism/d3d/D3DRTTexture.java

```diff
@@ -172,7 +172,7 @@ public boolean isOpaque() {
 
     @Override
     public boolean isVolatile() {
-        return getContext().isRTTVolatile();
+        return false;
```

Comment:
Quick note - this was checking on native side if we have non-Ex device and returning `true` if we did, `false` if D3D9 was initialized with Ex.

So far I only kept these changes within D3D backend. However, I checked the rest of the code and it seems that only D3D9 non-Ex ever returned anything else than `false`. Right now all the backends (d3d, egl, sw) return `false` in this method, which technically leaves a bit of dead code inside common part of Prism. Should we consider removing this method completely?

Reply:
> Should we consider removing this method completely?

Not as part of this PR. Maybe in a follow-up if you think it is worth doing as a (low priority) cleanup.